File: app/src/main/java/xyz/ashu/testssl/MyApplication.java
package xyz.ashu.testssl;

import android.app.Application;
import android.content.Context;

/**
 * <pre>
 * author : zhouyang
 * time   : 2019/08/21
 * desc   :
 * version:
 * </pre>
 */
public class MyApplication extends Application {
    private static Context mContext = null;

    @Override
    public void onCreate() {
        super.onCreate();
        mContext = this;
    }

    public static Context getContext() {
        return mContext;
    }
}

File: test_tinker/src/main/java/xyz/ashu/test_tinker/MyApplication.kt
package xyz.ashu.test_tinker

import android.app.Application
import com.tencent.bugly.Bugly

/**
 * <pre>
 * author : zhouyang
 * time   : 2020/05/04
 * desc   :
 * version:
 * </pre>
 */
class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        Bugly.init(this, "825e26ede6", true)
    }
}